packages/storage-ui/src/Components/Pages/BucketPage.tsx

```diff
@@ -160,9 +160,19 @@ const BucketPage: React.FC<IFileBrowserModuleProps> = () => {
 
   const handleUploadOnDrop = useCallback(async (files: File[], fileItems: DataTransferItemList, path: string) => {
     if (!bucket) return
+
+    if (accountRestricted) {
+      addToast({
+        type:"error",
+        title: t`Uploads disabled`,
+        subtitle: t`Oops! You need to pay for this month to upload more content.`
```

Comment:
Something in the line of "Please clear due payments to upload more content" like what we have in the restricted banner sounds more appropriate to me.

Reply:
What do you think about
```suggestion
        subtitle: t`Your account is restricted. Until you&apos;ve settled up, you can&apos;t upload any new content`
```

Reply:
Yup. This is better.

Reply:
Can you apply the same update to Files while you're at it?

Reply:
yup, that was my plan :D